### PR TITLE
Add blueprint to support Apptainer

### PR DESCRIPTION
**Pull request by wkharold**
This PR adds a blueprint to `community/examples` that builds a custom image with Apptainer installed.

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -13,60 +13,97 @@ md_toc github examples/README.md | sed -e "s/\s-\s/ * /"
...
-  * [tutorial-fluent.yaml](#tutorial-fluentyaml--) ![community-badge] ![experimental-badge]
-  * [omnia-cluster.yaml](#omnia-clusteryaml---) ![community-badge] ![experimental-badge] ![deprecated-badge]
-* [Blueprint Schema](#blueprint-schema)
-* [Writing an HPC Blueprint](#writing-an-hpc-blueprint)
-  * [Blueprint Boilerplate](#blueprint-boilerplate)
-  * [Top Level Parameters](#top-level-parameters)
-  * [Deployment Variables](#deployment-variables)
-  * [Deployment Groups](#deployment-groups)
-* [Variables and expressions](#variables-and-expressions)
+- [Example Blueprints](#example-blueprints)
```
Comment (mr0re1) [inline examples/README.md:16]: Thank you for addressing **TOC**, but we really like it more flat and with badges.
Please revert those changes and add your example here manually.

Comment (wkharold) [reply examples/README.md]: Done

Comment (mr0re1) [reply examples/README.md]: Looks like you last commit only removes single line. Could you please revert all changes to TOS , except addition of a single reference to a newly added section.

File: community/examples/hpc-slurm6-apptainer.yaml [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    source: community/modules/scheduler/schedmd-slurm-gcp-v6-controller
+    use:
+    - network
+    - compute_partition
+    - slurm_login
+    settings:
+      enable_controller_public_ips: true
+      disk_size_gb: $(vars.disk_size)
+      instance_image: $(vars.custom_image)
+      instance_image_custom: true
```
Comment (mr0re1) [inline community/examples/hpc-slurm6-apptainer.yaml:109]: Please fix:
```
109:34 [new-line-at-end-of-file] no new line character at the end of file
```

Comment (wkharold) [reply community/examples/hpc-slurm6-apptainer.yaml]: Done

File: community/examples/hpc-slurm6-apptainer.yaml [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
```
Comment (mr0re1) [inline community/examples/hpc-slurm6-apptainer.yaml:110]: Please fix:
```
yamllint.................................................................Failed
- hook id: yamllint
- exit code: 1

community/examples/hpc-slurm6-apptainer.yaml
  Error: 110:1 [trailing-spaces] trailing spaces
```

Comment (wkharold) [reply community/examples/hpc-slurm6-apptainer.yaml]: Done

**Comment by mr0re1** [comment]
@wkharold , there is a merge conflict, please resolve. Everything else LGTM.

File: examples/README.md [hunk 4]
```diff
@@ -514,7 +515,17 @@ Follow [hpc-slurm-tpu-maxtext].
 [hpc-slurm6-tpu-maxtext.yaml]: ../community/examples/hpc-slurm6-tpu-maxtext.yaml
 [hpc-slurm-tpu-maxtext]: ../docs/hpc-slurm6-tpu-maxtext.md
 
+<<<<<<< HEAD
```
Comment (mr0re1) [inline examples/README.md:518]: Artifacts of merge, please address.

File: examples/README.md [hunk 5]
```diff
@@ -514,7 +515,17 @@ Follow [hpc-slurm-tpu-maxtext].
...
+### [hpc-slurm6-apptainer.yaml] ![community-badge] ![experimental-badge]
+
+This blueprint creates a custom [Apptainer](https:https://apptainer.org) enabled image and builds an auto-scaling Slurm cluster using that image. You can deploy containerized workloads on that cluster as described [here](https://github.com/GoogleCloudPlatform/scientific-computing-examples/tree/main/apptainer).
+
+[hpc-slurm6-apptainer.yaml]: ../community/examples/hpc-slurm6-apptainer.yaml
+
+### [ml-slurm.yaml] ![core-badge]
+=======
 ### [ml-slurm-v5-legacy.yaml] ![core-badge]
+>>>>>>> develop
```
Comment (mr0re1) [inline examples/README.md:528]: same

File: community/examples/hpc-slurm6-apptainer.yaml [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
```
Comment (mr0re1) [inline community/examples/hpc-slurm6-apptainer.yaml:110]: Please address

```
110:1 [empty-lines] too many blank lines (1 > 0)
```

**Review by mr0re1 — APPROVED** [review]

**Pull request merged**